src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.tsx

```diff
@@ -123,39 +102,39 @@ class TodoItem extends React.Component {
         });
         break;
       default:
-        description = this.formatDescription();
+        description = this.formattedDescription;
     }
 
     const showSkipButton =
       task.skippable &&
       task.status !== 'skipped' &&
       task.status !== 'complete' &&
-      isExpanded &&
       !showConfirmation;
 
+    const Action = (p: React.HTMLAttributes<HTMLElement>) =>
+      task.actionType === 'external' ? (
+        <ActionExternalLink href={task.location} {...p} />
+      ) : (
+        <ActionTarget {...p} />
+      );
+
     return (
-      <Item
-        status={task.status}
-        onMouseOver={this.toggleDescription}
-        onMouseOut={this.toggleDescription}
-      >
+      <Item status={task.status}>
         <Content blur={showConfirmation}>
-          <Checkbox>{task.status && <IndicatorIcon status={task.status} />}</Checkbox>
-          <StyledLink
-            href={learnMoreUrl}
-            onClick={this.handleClick}
-            data-test-id={task.task}
-          >
+          <Action onClick={this.handleClick} data-test-id={task.task}>
+            <Checkbox status={task.status}>
+              {task.status && <IndicatorIcon status={task.status} />}
+            </Checkbox>
             <ItemHeader status={task.status}>{task.title}</ItemHeader>
-          </StyledLink>
-          <Description>{description}</Description>
-          <SkipButton
-            size="xsmall"
-            hide={!showSkipButton}
-            onClick={this.toggleConfirmation}
-          >
-            {t('Skip task')}
-          </SkipButton>
+            <Description>{description}</Description>
+            <SkipButton
```

Comment:
Are you able to skip with `SkipButton` inside `Action`?

Reply:
Yeah, made sure that works. Does some `preventDefault` magic.